### PR TITLE
Pre 4.26.2 Kotlin Clean Up

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

 - Updating some access modifiers to match Java semantics
 - Adding `@suppress` annotations to prevent docs generation for some library protected classes

 ### Checklist

 ~- [ ] Added a changelog entry~

- @sshropshire

### CODE REVIEW

File: BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt [hunk 1]
```diff
@@ -335,7 +335,7 @@ open class BraintreeClient @VisibleForTesting internal constructor(
      * @param activity
      * @return [BrowserSwitchResult]
      */
-    fun deliverBrowserSwitchResult(activity: FragmentActivity): BrowserSwitchResult? {
+    open fun deliverBrowserSwitchResult(activity: FragmentActivity): BrowserSwitchResult? {
```
Comment (KunJeongPark) [inline BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt:338]: Steven, what's the reason for these changes?

Comment (sshropshire) [reply BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt:338]: Java methods are `open` by default. With the exception of Java 8, we're still trying to adhere to semantic versioning when we can. We'll be able to remove all of the `open` modifiers once in our next major version.

**Review by KunJeongPark — APPROVED** [review]

**Review by jaxdesmarais — APPROVED** [review]
🚀

**Comment by sshropshire** [comment]
Calling out the source of these changes are from:

1. A local smoke test project I have set up that I'm hoping to create a repo for (just don't know where it should live)
2. A static analysis tool within Android Studio that highlights errors. I'm using this with a local build to make sure everything works in DropIn

Everything should be good now. There were a few gotchas that are now resolved.
